discuss/customizations/mail_external_confirm.rst

```diff
@@ -0,0 +1,52 @@
+===========================================================
+Confirmation on sending out messages to external recipients
+===========================================================
+
+Adding extra confirmation step while sending emails to external 
+recipients (partners, followers). This is included in the chatter 
+and advanced mail message composition wizard. Confirmation popover 
+will appear if email is going to be sent to any of external partners 
+with a list of external recipients.
+
+Configuration
+=============
+
+To start using this new functionality, module :code:`mail_external_confirm`
+must be installed.
+
+Feature can be enabled/disabled in General Settings 
+(:menuselection:`Settings --> General Settings --> Discuss`), Discuss 
+section by changing value of parameter 'Confirm External Emails':
+
+.. image:: media/mail_external_confirm_enable.png
+    :align: center
+
+Functionality
+=============
+
+This feature warns the user, that the message he/she is intended to send 
+contains any type of external recipients. It works in *Send Message*, 
+*Log Note* with @ or *Channels*. 
+
+This feature is active with *Leads*, *Opportunities* in CRM module and 
```

Comment:
Actually, feature will work on all objects (models) that have mail chatter implemented (depends on mail.thread module), so my suggestion would be to mention on what king of objects does this work and mention examples, e.g.:

'This feature will work on all objects that have mail chatter implemented, e.g. Leads & Opportunities, Partners, Sales, Invoices and etc.'